test/test_util.erl

```diff
@@ -0,0 +1,13 @@
+-module(test_util).
```

Comment:
In NextRoll, we usually call modules like this one `test_utils` (plural).
Sorry about this… but… do you mind changing its name?